recipe/meta.yaml

```diff
@@ -12,33 +11,57 @@ source:
 build:
   entry_points:
     - lightpath = lightpath.main:entrypoint
-  noarch: python
-  script: {{ PYTHON }} -m pip install . -vv
-  number: 0
+  number: 1
 
 requirements:
   host:
     - python >=3.9
-    - pip
-    - setuptools_scm
-  run:
-    - python >=3.9
-    - coloredlogs
-    - happi >=1.6.0
-    - numpy
-    - ophyd
-    - pcdsdevices >=3.4.0
-    - prettytable
-    - pydm
-    - pyqt >=5
-    - qtawesome
-    - qtpy
-    - typhos >=1.0.0
 
 test:
   imports:
     - lightpath
 
+outputs:
+  - name: lightpath-base
+    build:
+      noarch: python
+      script: {{ PYTHON }} -m pip install . -vv
+    requirements:
+      host:
+        - python >=3.9
+        - pip
+        - setuptools
+        - setuptools_scm
+      run:
+        - python >=3.9
+        - coloredlogs
+        - happi >=1.6.0
+        - ophyd
+        - prettytable
+      run_constrained:
+        - networkx <3.3  # networkx drops py3.9 support with 3.3
+        - numpy <2.0  # until upstream packages update (ophyd)
```

Comment:
These regressive pins are a bit worrying because they can lock us into old builds
I suggest removing both pins, but perhaps the networkx pin could stay with a python version filter? Unclear

Reply:
Let's try a pre-processing selector here